### ISSUE TITLE
Option to generate reports in .odt format

### ISSUE THREAD

**Issue by PaoloVecchi** [monarc-project/MonarcAppFO#128]
As you are all well aware both Private and Public Sectors ambition's is to free themselves from the yoke of proprietary formats and embrace the freedom provided by the Open Document Format so why not providing the option of saving odt files without proprietary fonts?

**Comment by cedricbonhomme** [comment]
Yes of course I agree. We should indeed change the fonts. Concerning the format of the templates it may be more difficult. We have to test (we checked already months ago).
